PaddleCV/object_detection/ppdet/data/data_feed.py

```diff
@@ -246,7 +246,8 @@ class SimpleDataSet(DataSet):
     def __init__(self,
                  dataset_dir=VOC_DATASET_DIR,
                  annotation=VOC_TEST_ANNOTATION,
-                 image_dir=VOC_IMAGE_DIR):
+                 image_dir=VOC_IMAGE_DIR,
+                 use_default_label=VOC_USE_DEFAULT_LABEL):
```

Comment:
shouldn't `VOC_USE_DEFAULT_LABEL` be renamed if it is not specific to pascal voc?

Reply:
Maybe all the defalut dir in SimpleDataSet should change.